content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -80,7 +80,7 @@ The following command will create a Horizontal Pod Autoscaler that maintains bet
 controlled by the php-apache deployment we created in the first step of these instructions.
 Roughly speaking, HPA will increase and decrease the number of replicas
 (via the deployment) to maintain an average CPU utilization across all Pods of 50%
-(since each pod requests 200 milli-cores by [kubectl run](https://github.com/kubernetes/kubernetes/blob/{{< param "githubbranch" >}}/docs/user-guide/kubectl/kubectl_run.md), this means average CPU usage of 100 milli-cores).
+(since each pod requests 200 milli-cores by [kubectl run](https://github.com/kubernetes/kubernetes/blob/feature-serverside-apply/docs/user-guide/kubectl/kubectl_run.md{{< param "githubbranch" >}}), this means average CPU usage of 100 milli-cores).
```

Comment:
```suggestion
(since each pod requests 200 milli-cores by kubectl run), this means average CPU usage of 100 milli-cores).
```
The link target is a blank document with the statement that it's autogenerated. In this case I'd just remove the link altogether. It looks like the link target hasn't contained anything since 2016. 😬